### ISSUE TITLE
Error Report

### ISSUE THREAD

**Issue by cf7180560** [FelisCatus/SwitchyOmega#141]
SwitchyOmega 2.2.9
Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36
```
[5] Unhandled rejection:
 Error: SystemProfile cannot be used in PAC scripts
    at Object.module.exports.exports._profileTypes.SystemProfile.compile (chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/omega_pac.min.js:13801:15)
    at Object.module.exports.exports.compile (chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/omega_pac.min.js:13779:45)
    at chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/omega_pac.min.js:13263:29
    at Object.module.exports.script (chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/omega_pac.min.js:13267:9)
    at ChromeOptions.Options.pacForProfile (chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/omega_target.min.js:10764:33)
    at ChromeOptions.applyProfileProxy (chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/omega_target_chromium_extension.min.js:3611:27)
    at ChromeOptions.Options.applyProfile (chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/omega_target.min.js:10884:19)
    at chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/background.js:339:23
From previous event:
    at chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/background.js:338:25
From previous event:
    at new Promise (chrome-extension://padekgcemlokbadohgkifijomclgjgif/js/omega_target.min.js:1972:37)
    at CustomBindingsObject.getAsync (chrome-extension://padekg

**Comment by FelisCatus** [comment]
Please upload the backup file to https://gist.github.com and paste the link here.

**Comment by FelisCatus** [comment]
Closing this issue for now. Please attach the backup file if you want to reopen it.

**Comment by FelisCatus** [comment]
The fix was just released with v2.3.14.
